### PR TITLE
Teams 2.1 Images Updates

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
1. App Complete Sample - **Nodejs**
2. App Hello World - **Csharp and Nodejs** 
3. app-link-unfurling-in-share-to-teams - **Csharp and Nodejs**

### CODE REVIEW

File: samples/meetings-notification/nodejs/Images/4.SubmitFeedback.png
Comment (Pawank-MSFT) [inline samples/meetings-notification/nodejs/Images/4.SubmitFeedback.png:1]: Please check this Task module CSS

**Review by Pawank-MSFT — COMMENTED** [review]
Please fix small comments

File: samples/meetings-notification/nodejs/Images/4.SubmitFeedback.png
Comment (Mohammed-MSFT) [reply samples/meetings-notification/nodejs/Images/4.SubmitFeedback.png:1]: PR Comments Fixed

File: samples/app-complete-sample/nodejs/images/5.MultiDialog.png
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/app-complete-sample/nodejs/images/5.MultiDialog.png:1]: There is No Difference in Old Image and New Image
**Old Image From Repo (Half Picture Uploaded)**
image

**New Image Teams 2.1 Full Picture uploaded**
5 MultiDialog

**Review by Pawank-MSFT — APPROVED** [review]
Looks correct, Approving!

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Pawank-MSFT** [comment]
Executed build pipeline : 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8890540462

**Pull request merged**